packages/api/src/api/extras/powerLevel.test.ts

```diff
@@ -38,8 +76,8 @@ describe('isValidPowerLevelStateEvent', () => {
       },
       event_id: 'event-id',
       origin_server_ts: 0,
-      room_id: 'room-id',
-      sender: 'user-id',
+      room_id: '!room-id:example.com',
```

Comment:
Seems like would make sense to move the test for `isValidPowerLevelStateEvent` to `events.test.ts`, because 'isValidPowerLevelStateEvent' is not in events.